docs/UserGuide.md

```diff
@@ -219,7 +219,7 @@ All fields are not subjected to the restriction in the input table and can take
 ### Feature: Candidate Management
 
 Manage a list of candidates for your company, with the simple instructions below!
-Each candidate is uniquely identified their email. 
+Each candidate is uniquely identified by their email.
 Different candidates can have the same name as along as the name, email and phone are not all the same.
```

Comment:
There is a conflict here. Master branch implementation uses only email as the unique identifier.